OpenSim/Common/Component.h

```diff
@@ -1996,10 +2136,34 @@ void ComponentListIterator<T>::advanceToNextValidComponent() {
 
 template<class C>
 void Connector<C>::findAndConnect(const Component& root) {
-    const C& comp = root.getComponent<C>(get_connectee_name());
-    connectee = comp;
+ 
+    const std::string& path = get_connectee_name();
+    const C* comp = nullptr;
+
+    try {
+        if (path[0] == '/') { //absolute path name
```

Comment:
We should (eventually) have a central place for making queries about a path so that this check could be `ComponentPathUtil::isAbsolute(path)` or something like that. Or, we can have a new class, "ComponentPath"; then we could do `path.isAbsolute()`. Not for this PR...